### PR TITLE
simplified Session instantiation

### PR DESCRIPTION
**Pull request by jakubczakon**
* `Session` can be now instantiated directly via the `api_token`
* updated docstrings
* updated tests

### CODE REVIEW

File: neptunelib/session.py [hunk 1]
```diff
@@ -25,22 +25,33 @@ class Session(object):
     In order to query Neptune experiment database in any way you have to instantiate this object first.
 
     Attributes:
-        credentials (:obj:`Credentials`): Description of `attr2`.
+        api_token(str): This is a secret API key that you can retrieve by running `neptune account api-token get <org>`.
```
Comment (pitercl) [inline neptunelib/session.py]: It's actually `neptune account api-token get` (no `<org>`). The docstring was inaccurate.

File: neptunelib/session.py [hunk 2]
```diff
@@ -25,22 +25,33 @@ class Session(object):
...
 
     Args:
+        api_token(str): This is a secret API key that you can retrieve by running `neptune account api-token get <org>`.
```
Comment (pitercl) [inline neptunelib/session.py]: Same here.

File: neptunelib/session.py [hunk 3]
```diff
@@ -25,22 +25,33 @@ class Session(object):
...
+
+        >>> session = Session()
     """
 
-    def __init__(self, credentials=None):
-        self.credentials = credentials or Credentials.from_env()
+    def __init__(self, api_token=None):
+        credentials = Credentials(api_token) if api_token else Credentials.from_env()
+
+        self.api_token = api_token
```
Comment (pitercl) [inline neptunelib/session.py]: Do we need this variable?

Comment (jakubczakon) [reply neptunelib/session.py]: I don't think we do -> dropping it.

**Pull request merged**